src/Beberlei/Bundle/MetricsBundle/DependencyInjection/Configuration.php

```diff
@@ -37,6 +37,8 @@ public function getConfigTreeBuilder()
                             ->scalarNode('source')->defaultNull()->end()
                             ->scalarNode('username')->defaultNull()->end()
                             ->scalarNode('influxdb_client')->defaultNull()->end()
+                            ->scalarNode('prometheus_collector_registry')->defaultNull()->end()
```

Comment:
you need to add a validation rule making sure that this one is not empty for ``prometheus`` collectors (see below how it is done for librato validation)